src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -212,7 +212,7 @@ private long countTasksOverdue(List<Task> tasks) {
    * @return List of tasks from all task lists in user's account
    * @throws IOException if an issue occurs with the tasksService
    */
-  private List<Task> getAllTasksFromAllTaskLists(TasksClient tasksClient) throws IOException {
+  public static List<Task> getAllTasksFromAllTaskLists(TasksClient tasksClient) throws IOException {
```

Comment:
If you also need this I agree this can be public and static, but this should probably be in the TasksClient

Reply:
Agreed, this is used in the Go servlet as well so this would be perfect - not sure why I did not do this in the first place. Thanks!

Reply:
`RESOLVED`